config/locales/devise.en.yml

```diff
@@ -9,7 +9,7 @@ en:
     failure:
       already_authenticated: "You are already signed in."
       inactive: "Your account is not activated yet."
-      invalid: "Invalid %{authentication_keys} or password."
+      invalid: "Invalid email or password."
```

Comment:
thi no lay authentication_key la Email nhung la` chu in hoa... doi lai chu thuong cho dep :))